packages/storybook/stories/MessageThread/MessageThread.stories.tsx

```diff
@@ -402,6 +404,10 @@ const MessageThreadStory = (args): JSX.Element => {
     setChatMessages([...chatMessages, GenerateMockNewChatMessageWithInlineImage()]);
   };
 
+  const onSendNewMessageWithFileAttachment = (): void => {
```

Comment:
```suggestion
  const onSendNewMessageWithAttachment = (): void => {
```

Reply:
This PR focus on the FileCard migration to FluentUI, API will be re-worked by John

Reply:
I don't think this is part of the public API?
Let's get it updated here, as it looks like it got missed during the File -> Attachment renaming PR.